### PR TITLE
chore: bump latest version to 1.00.0-beta

### PR DESCRIPTION
**Pull request by alagane**
Use version 1.00.0-beta (also 0.27 and latest)
Modifiy and correct parameters names to add an user

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,16 +37,16 @@ Versions match. As an example, to get Petals Cockpit 0.20.0, just type in `docke
...
+* user: `admin`
 * password: `admin`
 
 It is possible to change this user with the following parameters: 
 
 | Argument | Optional | Default | Description |
 | -------- | :------: | :-----: | ----------- |
-| COCKPIT_USER | yes | `admin` | The user's id, also his login. |
-| COCKPIT_USERNAME | yes | `admin` | The name under which the user will appear. |
+| COCKPIT_USERNAME | yes | `admin` | The user's id, also his login. |
```
Comment (vincent-zurczak) [inline README.md:48]: **USER_ID** would be more explicit then.

File: README.md [hunk 2]
```diff
@@ -37,16 +37,16 @@ Versions match. As an example, to get Petals Cockpit 0.20.0, just type in `docke
...
+| COCKPIT_NAME | yes | `admin` | The name under which the user will appear. |
```
Comment (vincent-zurczak) [inline README.md:49]: Or you could consider using **COCKPIT_USER_DISPLAY_NAME** for this one.

**Review by vincent-zurczak — CHANGES_REQUESTED** [review]
I expect your answers before merging. :wink:

File: README.md (hunk 1 above)
Comment (psouquet) [reply README.md:48]: `username` is the term used in cockpit for the user UID.

File: README.md (hunk 2 above)
Comment (psouquet) [reply README.md:49]: `name` is the term used in cockpit for the user display name. But I guess `COCKPIT_USER_DISPLAY_NAME` is close enough and clearer.

**Pull request merged**